docs/wiki/setup-google-cloud-account.md

```diff
@@ -0,0 +1,35 @@
+# Set up a Google Cloud account
```

Comment:
```suggestion
# Set Up a Google Cloud Account
```

Reply:
Probably should standardise case for titles in the docs someday. 

Fixed in next commit